framework/encode/capture_manager.cpp

```diff
@@ -1322,7 +1349,11 @@ void CommonCaptureManager::ActivateTrimming(std::shared_lock<ApiCallMutexT>& cur
             exclusive_api_call_lock = AcquireExclusiveApiCallLock();
         }
 
-        capture_mode_ |= kModeWrite;
+        capture_mode_ |= kModeTrim;
```

Comment:
This is new.  Should it be an assert verifying the value was already set?  Otherwise we are probably here for a wrong reason, right?